sdk/helper/keysutil/lock_manager.go

```diff
@@ -59,6 +59,10 @@ type PolicyRequest struct {
 
 	// AllowImportedKeyRotation indicates whether an imported key may be rotated by Vault
 	AllowImportedKeyRotation bool
+
+    // NOTE: Desc
+    // IsCiphertextSet indicates an import is for a private or public key
+    IsCiphertextSet bool
```

Comment:
This makes sense at a API request handling level, but I don't think it makes sense here.

Maybe, `IsPublicKeyOnly`?